server/db/models/students.js

```diff
@@ -30,7 +30,7 @@ const Student = db.define(
   },
   {
     hooks: {
-      beforeCreate: student => {
+      beforeCreate: (student) => {
```

Comment:
Since we don't teach this syntax for hooks, I think we should be consistent and follow [method 3 of declaring hooks](https://sequelize.org/master/manual/hooks.html#declaring-hooks)

Reply:
Remnants. Updated!